autoload/simplenote.vim

```diff
@@ -728,49 +723,75 @@ endfunction
 
 function! simplenote#SimpleNote(param, ...)
 python << EOF
-param = vim.eval("a:param")
-optionsexist = True if (float(vim.eval("a:0"))>=1) else False
-if param == "-l":
-    if optionsexist:
+def reset_user_pass(warning=None):
+    vim.command("let s:user=''")
+    vim.command("let s:password=''")
+    if warning:
+        vim.command("redraw!")
+        vim.command("echohl WarningMsg")
+        vim.command("echo '%s'" % warning)
+        vim.command("echohl none")
+
+def Simplenote_cmd():
+    if vim.eval('s:user') == '' or vim.eval('s:password') == '':
         try:
-            # check for valid date string
-            datetime.datetime.strptime(vim.eval("a:1"), "%Y-%m-%d")
-            interface.list_note_index_in_scratch_buffer(since=vim.eval("a:1"))
-        except ValueError:
-            interface.list_note_index_in_scratch_buffer(tags=vim.eval("a:1").split(","))
-    else:
-        interface.list_note_index_in_scratch_buffer()
-
-elif param == "-d":
-    interface.trash_current_note()
+            vim.command("let s:user=input('username:', '')")
```

Comment:
Simplenote calls this "Email" on their website & app, so I think that should be used instead of "username".